RELEASES.md

```diff
@@ -1,11 +1,28 @@
 # Releases
 
 > Note: ES3/IE8 compatibility will be removed in the future v3.x.x releases (scheduled for mid-late 2022), so if you need to retain ES3 compatibility you will need to remain on the 2.x.x versions of the SDK or your runtime will need install polyfill's to your ES3 environment before loading / initializing the SDK.
+## 2.8.4 (Jun 1st, 2022)
+
+
+- Updates React Plugin to v3.3.4 (with v2.8.4 as dependency) -- using React 17
+- Updates React Native Plugin to 2.5.34 (with v2.8.4 as dependency)
+- Updates Chrome Debug Extension to 0.3.4
+
+### Changelog
+
+- #198 Run-time Telemetry initializers for Ajax requests
```

Comment:
Looks like we are using PR titles for change log, but some of them look a little weird like questions in 1776 or task numbers, branch names that doesn't mean much here, it will be good to be more descriptive about what the change is about and remove extra not relevant details

Reply:
These are a combination of Issue names and PR names, I try to make them more specific when I know more details about them -- but don't always achieve that goal.

Reply:
By having the # number in the markdown file GitHub turns these into links directly to the Issue or PR for more context.